js/ui/calendar/ui.calendar.js

```diff
@@ -1156,11 +1261,17 @@ const Calendar = Editor.inherit({
         delete this._viewWidthValue;
     },
 
+    _clearInlineWidth: function() {
+        this.$element().css('width', '');
+    },
+
     _disposeViews: function() {
         this._view.$element().remove();
```

Comment:
[this._view, this._additionalView, this._beforeView, this._afterView].forEach(view => {
    view?.$element.remove();
    delete view;
})

Reply:
Looks like we cannot delete object fields in forEach

![image](https://user-images.githubusercontent.com/29150145/226730616-9a17098c-caee-42e8-9a90-7d6f20f11cfc.png)